### ISSUE TITLE
Look at having always ON webserver on localhost

### ISSUE THREAD

**Issue by koying** [koying/SPMC#271]

**Comment by MrMC** [comment]
json also works via tcpserver. you could create a special instance on localhost for leanback usage. Much cleaner and less intrusive than dealing with webserver.

**Comment by koying** [comment]
Instance of webserver? That's a possibility.
I need a webserver to download the thumbs and fanarts...

See https://github.com/Razzeee/xbmc/pull/2 for the full list of (fugly, imo) alternate solutions ;)

**Comment by koying** [comment]
BTW, did you considered http://developer.android.com/reference/android/content/res/AssetManager.html before deciding to go the direct way?

**Comment by MrMC** [comment]
yep, AssetManager means a hop through jni into java and back. not the fastest thing in the world.

**Comment by MrMC** [comment]
I glanced at Razzeee/xbmc#2 , stopped after the 20th 'no' :)

**Comment by koying** [comment]
Hehe... He just did a big merge and asked me to mark the changes unrelated to leanback ;)

**Comment by koying** [comment]
Ah damn, commented on the wrong ticket. the AssetManager thing relates to the apk vfs, obviously ;)
